<launch>
  <node pkg="tf" type="static_transform_publisher" name="slow_tf" args="0 0 0 0 0 0 base_link slowzone 100"/>
  <node pkg="tf" type="static_transform_publisher" name="stop_tf" args="0 0 0 0 0 0 base_link stopzone 100"/>

  <node name="odomListener" pkg="ti_safety_bubble" type="odomListener" output="screen" >
    <param name="bubble_shape" value="circular" />
    <param name="slow_radius" value="2.0" />
    <param name="stop_radius" value="1.0" />
    <param name="clear_costmap_period_secs" value="5" />
  </node>

  <node name="orientation" pkg="ti_safety_bubble" type="orientation" output="screen" >
  </node>
</launch>
